build.gradle

```diff
@@ -133,7 +133,7 @@ ext {
     androidxWorkVersion = "2.4.0"
 
     daggerVersion = '2.29.1'
-    fluxCVersion = '1.21.0-beta-6'
+    fluxCVersion = '1.21.0'
```

Comment:
Side note: I realized after submitting the 17.8-rc-1 beta that I didn't `git push` the commit that updated this before triggering the beta, which means the 17.8-rc-1 beta that got sent to beta-testers was build while pointing to `1.21.0-beta-6`. That's why the `17.8-rc-1` tag points to 165f1ec46acde40e5a8ea6f099d89928e62d5173 instead of 2f5ca88dd13aaa53282a76eace4a6b7d38129029 (which updates this line).

But that's not a problem because it happens that FluxC's `1.21.0-beta-6` and `1.21.0` tags both point to the exact same commit (and currently `develop`'s tip), so that doesn't affect the end result, which is why I decided not to redo a new beta build just for that.